front/app/containers/Admin/sideBar/messages.ts

```diff
@@ -57,12 +57,20 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SideBar.insights',
     defaultMessage: 'Reporting',
   },
+  academy: {
+    id: 'app.containers.AdminPage.SideBar.academy',
+    defaultMessage: 'Academy',
+  },
+  linkToAcademy: {
+    id: 'app.containers.AdminPage.SideBar.linkToAcademy',
+    defaultMessage: 'https://academy.citizenlab.co/',
```

Comment:
If this will be the same URL for everybody I'm not sure it needs to be entered in the translation system. It looks like the Academy only has EN and NL languages for now so I'm not sure there would even be content for the rest of the languages

Reply:
Ahh cool, you already had this conversation. I also would assume that `https://community.citizenlab.co/${currentLocale}/folders/guide` makes more sense but it seems like Crowdin translation is more flexible, so all good